core-bundle/contao/forms/Form.php

```diff
@@ -218,7 +218,11 @@ protected function compile()
 
 				if ($objWidget instanceof UploadableWidgetInterface)
```

Comment:
```suggestion
				if ($objWidget instanceof UploadableWidgetInterface && $objWidget->value)
```

Reply:
No, we cannot, because then `$hasUpload = true;` will not be executed and consequently the wrong `enctype` will be output in the front end for example.

Reply:
Ah I see!